medusa/indexers/indexer_config.py

```diff
@@ -114,3 +118,40 @@
         'identifier': 'tmdb',  # Also used as key for the custom scenename exceptions. (_get_custom_exceptions())
     }
 }
+
+
+def create_config_json(indexer):
+    """Create a json (pickable) compatible dict, for using as an apiv2 resource."""
+    return {
+        'enabled': indexer['enabled'],
+        'id': indexer['id'],
+        'name': indexer['name'],
+        'apiParams': {
+            'language': indexer['api_params']['language'],
+            'useZip': indexer['api_params']['use_zip'],
+            'session': '',
```

Comment:
Is `session` needed if it's always empty?

Reply:
no we can remove that